src/WinRT.Runtime/ComWrappersSupport.cs

```diff
@@ -334,6 +337,22 @@ internal static object GetRuntimeClassCCWTypeIfAny(object obj)
             });
 
             return entries;
+
+            void AddInterfaceToVtable(Type iface)

```

Comment:
I know this method is not annotated, but is part is trim-safe yet? I'm seeing eg. some warnings in `FindHelperType` and whatnot, which are just not surfaced here because the method itself is not annotated. Just trying to better understand which parts are "ok" already and which are still work in progress 🙂

Reply:
I expect there to be changes to both FindHelperType and GetAbiToProjectionVftblPtr as I work through my generic changes.